servlet/application.properties

```diff
@@ -95,7 +95,7 @@ suit.attivio.password=attivio
 # or the main HTML file. Generally, the list of routes here should match the
 # routes defined in your application's main component (e.g. SearchUIApp.js in the
 # out-of-the-box version). It is a comma-separated list (with no spaces).
-suit.attivio.routes=/,/landing,/results,/results-no-mast,/insights,/doc360,/login,/error
+suit.attivio.routes=/,/landing,/results,/results-no-mast,/insights,/doc360,/login
```

Comment:
We should make a corresponding change to remove `/error` from the default values used when this isn't set in the properties file. See the `DEFAULT_ROUTES` constant in `HomeControllerHandlerMapper.java`...